CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# How to Contribute
+
+Information on contributing to this repo is in the [Contributing Guide](https://github.com/AndcultureCode/AndcultureCode/blob/master/CONTRIBUTING.md) in the Home repo.

```

Comment:
Nitpick - `master` to `main` (We should probably go back and update these across the board... Must have missed them in our initial rename issues)